tests/tests_integration/test_api/test_relationships.py

```diff
@@ -156,6 +157,13 @@ def test_retrieve_unknown(self, new_relationship):
             API_REL.retrieve_multiple(external_ids=["this does not exist"])
         assert API_REL.retrieve(external_id="this does not exist") is None
 
+    def test_update_single(self, new_relationship):
+        new_rel, ext_id = new_relationship
+        updated_rel = API_REL.update(RelationshipUpdate(ext_id).target_type.set("timeseries").confidence.set(None))
+        assert isinstance(updated_rel, Relationship)
+        assert updated_rel.dump()["target_type"] == "timeSeries"
```

Comment:
why dump?

Reply:
Copypasta from an above test. Changed here and above.